Blockzilla/BrowserViewController.swift

```diff
@@ -228,7 +228,12 @@ class BrowserViewController: UIViewController {
     override func viewWillAppear(_ animated: Bool) {
         navigationController?.setNavigationBarHidden(true, animated: animated)
         
-        homeView?.setHighlightWhatsNew(shouldHighlight: shouldShowWhatsNew())
+        if let homeViewToolset = homeView?.toolbar.toolset {
+            homeViewToolset.setHighlightWhatsNew(shouldHighlight: homeViewToolset.shouldShowWhatsNew())
+            homeView?.toolbar.layoutIfNeeded()
```

Comment:
Is ```homeView?.layoutIfNeeded()``` good enough here? Or do we need to specifically layout the toolbar? Doing the former would allow us to make toolbar private.